experimental/devices/ina219/ina219.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package ina219
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/mmr"
+	"periph.io/x/periph/conn/physic"
+)
+
+// Config is a Configuration
+type Config struct {
+	Address       int
+	SenseResistor physic.ElectricResistance
+	MaxCurrent    physic.ElectricCurrent
+}
+
+const (
```

Comment:
Instead, create a DefaultOpts variable that people can use directly. Here's how it looks in practice:
https://periph.io/x/periph/devices/bmxx80/#pkg-variables
https://periph.io/x/periph/devices/cap1xxx#pkg-variables